apps/docs/scripts/typedoc-postbuild.ts

```diff
@@ -65,7 +65,12 @@ const exportLinksJson = () => {
     .forEach((directory) => {
       links.items.push({ text: directory, link: `/api/${directory}/`, collapsed: true, items: [] });
       readdirSync(join(apiDocsDir, directory))
-        .filter((file) => file !== 'index.md')
+        .filter(
```

Comment:
Pardon my ignorance as I'm not super familiar with this postbuild script but I do get the impression we probably should do a wider refactor on it. 

I think we can make this a bit more flexible though should someone want to add other file names within multiple entry points

```suggestion
.filter((file) => {
          // Exclude index files and files related to secondary entry points
          const isIndexFile = file.endsWith('index.md');
          const isSecondaryEntryPoint = secondaryEntryPoints.some((entryPoint) =>
            file.includes(entryPoint.replace('-', '-').replace('.md', ''))
          );
          return !isIndexFile && !isSecondaryEntryPoint;
        })
```

Reply:
I do agree, although I think it's been simplified heavily with the new typedoc version as in #2650.

Reply:
I'm personally wondering whether we want to move the API docs out of the docs hub and hosting this elsewhere. As it adds a lot of noise to the [docs](https://docs.fuel.network/docs/fuels-ts/api/).

Reply:
I agree tbh, the default typedoc styling is really [clean](https://typedoc.org/example/) and it bugs me some time that the docs hub search includes all the api doc links.